ibis/pandas/udf.py

```diff
@@ -216,41 +216,58 @@ def pre_execute_analytic_and_reduction_udf(op, *clients, scope=None, **kwargs):
     input_type = op.input_type
     nargs = len(input_type)
 
-    # An execution rule for a simple aggregate node
+    # An execution rule to handle analytic and reduction UDFs over
+    # 1) an ungrouped window,
+    # 2) an ungrouped Aggregate node, or
+    # 3) an ungrouped custom aggregation context
     @execute_node.register(type(op), *(itertools.repeat(pd.Series, nargs)))
-    def execute_udaf_node(op, *args, **kwargs):
-        return op.func(*args)
+    def execute_udaf_node_no_groupby(op, *args, **kwargs):
```

Comment:
```
def execute_udaf_node_no_groupby(op, *args, aggcontext, **kwargs):
```
Does this work instead of poping out `aggcontext` from kwargs?

Reply:
This does work as well. I've made this change